### PR TITLE
Add New Fun Fact

### PR DESCRIPTION
**Pull request by Dhuls-Scratch**
haven't tested it, because i'm so lazy, but i don't think it will break anything

### CODE REVIEW

File: components/Loading.vue [hunk 1]
```diff
@@ -56,6 +56,7 @@ export default {
         "the",
         "ocular uses vue",
         "no",
+        "the ice cream machine is broken",
```
Comment (FunctionalMetatable) [inline components/Loading.vue:59]: maybe add mcdonalds

**Review by ScolderCreations — APPROVED** [review]
this would be a nice dyk

**Comment by Dhuls-Scratch** [comment]
ok it wasn't conflicting when this was first made.

**Comment by return-misaPuding** [comment]
IDK, something broke all these fun fact PRs :(

**Comment by ScolderCreations** [comment]
> IDK, something broke all these fun fact PRs :(

Probably my own PR.

**Comment by JaydenDev** [comment]
One time I made a PR adding a loading message it was never considered :(

**Comment by JaydenDev** [comment]
#123

**Comment by derter-menter** [comment]
I do not like "joke" facts, they are uninteresting. This is not even funny

**Comment by jeffalo** [comment]
thanks, but we really don't need any more loading screen tips right now.

**Pull request closed without merging**